Palaso.Tests/PlatformUtilities/PlatformTests.cs

```diff
@@ -4,23 +4,19 @@
 using NUnit.Framework;
 using Palaso.PlatformUtilities;
 
-// For code shipped with products, please try to use runtime checks to select code to run.
-// There are cases that require compile-time conditionals (e.g. when testing the runtime or
-// referencing platform specific imported assemblies like MonoMac.dll).  In this case, do
-// The following:
-// 1) include the following PropertyGroup in the .csproj after all of the DefineConstants
-//    elements.  [Note: _system_name and _system_type are defined by xbuild in later 
-//    versions of Mono.  We tested with Mono 3.12 that comes with Xamarin Studio for 
-//    Mac.  To verify, add /verbosity:diag to xbuild command-line. YMMV.]
+// For code shipped with products, please try to use runtime checks to select 
+// code to run.  There are cases that require compile-time conditionals 
+// (e.g. when testing the runtime or referencing platform specific imported 
+// assemblies like MonoMac.dll).  In this case, do The following:
+// 
+// 1) Include the following at the bollow of the csproj file
```

Comment:
Actually, this shouldn't be here anymore.  It is documented in platform.targets.